### PR TITLE
added basic comments for methods

### PR DESCRIPTION
**Pull request by thejoshuahendrix**
If you have any suggestions where we can add comments like these, or how to improve the quality of these comments lmk!

### CODE REVIEW

**Comment by thejoshuahendrix** [comment]
Also did a small change to a map method, replacing k with item, inferring item as in row item since we are mapping from a row, but if you prefer the shorthand just change it back

**Comment by lucemans** [comment]
> Also did a small change to a map method, replacing k with item, inferring item as in row item since we are mapping from a row, but if you prefer the shorthand just change it back

Yes, My apologies, when initially writing Scyllo a lot of ninja code made its way in, if you have any other contributions to fix some of the naming etc, feel free to!

File: src/ScylloClient.ts [hunk 1]
```diff
@@ -45,11 +45,17 @@ export class ScylloClient<TableMap extends Tables> {
         this.debug = options.debug || false;
         this.log = options.log || console.log;
     }
-
+    /**
+     * Connect
+     * to the client.
+     */
```
Comment (lucemans) [inline src/ScylloClient.ts:51]: I do have to mention that technically the `cassandra-driver` library we are using underneath automatically connects to the servers. So technically this only really holds until that connection is established, but technically this doesnt "trigger a connet" more so then "wait till connection is done"

File: src/ScylloClient.ts [hunk 2]
```diff
@@ -23,15 +23,15 @@ export type ScylloClientOptions = {
     log?: LogMethod;
     /**
      * Verbose,
-     * Wether or not to output verbose information on each request.
+     * Whether or not to output verbose information on each request.
```
Comment (lucemans) [inline src/ScylloClient.ts:26]: I-... we not gonna talk about my shitty english xD

File: src/ScylloClient.ts (hunk 1 above)
Comment (lucemans) [reply src/ScylloClient.ts:51]: Could you find a way to sort-of clarify that? Cuz I have no clue

**Review by lucemans — CHANGES_REQUESTED** [review]

**Review by lucemans — APPROVED** [review]
Hell yeah!

**Pull request merged**